doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+*Height of thumbnails generated from uploaded images. Updating this value changes how thumbnail images render in future, but does not change images created in the past.*
+
+```"PreviewWidth": 1024```
+*Maximum width of preview image. Updating this value changes how preview images render in future, but does not change images created in the past.*
+
+```"PreviewHeight": 0```
+*Maximum height of preview image ("0": Sets to auto-size). Updating this value changes how preview images render in future, but does not change images created in the past.*
+
+```"ProfileWidth": 128```
+*Width of a profile picture recommended when uploading from Account Settings. A users full size profile picture is shown when you hover over their name in a channel thread.*
+
+```"ProfileHeight": 128```
+*Height of a profile picture recommended when uploading from Account Settings. A users full size profile picture is shown when you hover over their name in a channel thread.*
+
+```"EnablePublicLink": true```
+*”true”: Allow users to share public links to files and images when previewing. “false”*
+
+```"PublicLinkSalt": "A705AklYF8MFDOfcwh3I488G8vtLlVip"```
+*32-character (to be randomly generated via Admin Console) salt added to signing of public image links.*
+
+
+### Log Settings
+___
+Settings to configure the console and log file output, detail level, format and location of error messages.
+
+### Console Settings
+
+```"EnableConsole": true```
+*“true”: Output log messages to the console based on ConsoleLevel option. The server writes messages to the standard output stream (stdout).*
+
+```"ConsoleLevel": "DEBUG"```
+*Level of detail at which log events are written to the console when EnableConsole=true. ”ERROR”: Outputs only error messages; “INFO”: Outputs error messages and information around startup and initialization; “DEBUG”: Prints high detail for developers debugging issues.*
+
+### Log File Settings
+
+```"EnableFile": true```
+*”true”:  Log files are written to files specified in FileLocation.*
+
+```"FileLevel": "INFO"```
+*Level of detail at which log events are written to log files when EnableFile=true. “ERROR”: Outputs only error messages; “INFO”: Outputs error messages and information around startup and initialization; “DEBUG”: Prints high detail for developers debugging issues.*
+
+```"FileFormat": ""```
+*Format of log message output. If blank, FileFormat = "[%D %T] [%L] (%S) %M", where: 
+  
+    %T		Time (15:04:05 MST) 
+    %t		Time (15:04) 
+    %D		Date (2006/01/02) 
+    %d		Date (01/02/06) 
+    %L		Level (FNST, FINE, DEBG, TRAC, WARN, EROR, CRIT) 
+    %S		Source 
+    %M		Message*
+
+```"FileLocation": ""```
+*Directory to which log files are written. If blank, log files write to ./logs/mattermost, which writes logs to mattermost.log. Log rotation is enabled and every 10,000 lines of log information is written to new files stored in the same directory, for example mattermost.2015-09-23.001, mattermost.2015-09-23.002, and so forth.*
```

Comment:
I think this "If blank, log files write to ./logs/mattermost, which writes logs to mattermost.log" can just be "If blank, log files write to ./logs/mattermost/mattermost.log"